app/scripts/modules/core/src/presentation/forms/fields/TextField.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+import { BasicLayout } from '../layouts';
+import { formikField, IFormikField } from './formikField';
+import { PartialInputProps, IFieldLayoutProps, IFieldProps } from '../interface';
+
+export type ITextFieldProps = IFieldProps & PartialInputProps;
+export class TextField extends React.Component<ITextFieldProps> {
+  static Formik: IFormikField<ITextFieldProps> = formikField<ITextFieldProps>(TextField);
```

Comment:
does it use the browser default styling? 
If we use this: https://github.com/spinnaker/styleguide/blob/master/src/styles/forms.css#L12, and add the selector sp-form to the container of this field, do formik styles override this?

Reply:
we discussed this in person, but putting here for posterity:  formik doesn't have styles